adm/src/database/lmdb.rs

```diff
@@ -0,0 +1,161 @@
+use std::collections::HashMap;
```

Comment:
Suggest adding api docs to the database code.

Reply:
I want to finalize this API before adding docs, which won't happen until I finish STL-1007, STL-1008, and STL-1009. I also want to create a Database trait that is implemented and used, which may influence this API.

Reply:
ok